bin/rename-skill

```diff
@@ -29,18 +32,51 @@ console.log(
   chalk.magenta(util.inspect(newSkillNames))
 )
 
-const renameSkillsWithinModules = () =>
+const renameSkillsWithinContainer = (container) =>
   generateDigest().then(digest => {
     const skillId = nameToId(currentSkillName)
-    const modules = Object.values(digest.modules).filter(module =>
-      module.skills.includes(skillId)
-    )
+    const containerItems = Object.values(digest[container]).filter(containerItem => {
+
+      const containerValues = Object.values(containerItem)
+
+      let temp
+      if(container === 'modules') {
+        temp = containerItem.skills.includes(skillId)
+      } else if(container === 'skills') {
+        temp = containerValues.includes(skillId)
+      }
+
+      return (temp)
+    })
 
     return Promise.all(
-      modules.map(renameSkillsWithinModule)
+      containerItems.map({
+        modules: renameSkillsWithinModule,
+        skills: renameSkillsWithinMarkdown,
+      }[container])
     )
   })
 
+const renameSkillsWithinMarkdown = (skill) => {
+  const oldMarkdownPath = path.join(__dirname, '..', skill.markdownFilePath)
+  const newMarkdownPath = path.join(
+    __dirname,
+    '..',
+    'skills'
+  ) + '/' + newSkillNames[0].replace(/[\'\,\/\s]/g, '-') + '.md'
+
+  return fs.readFile(oldMarkdownPath)
+    .then(buffer => buffer.toString())
+    .then(rawtext => {
+      const newText = rawtext.replace(/\#.*\n/, `# ${newSkillNames[0]}\n`)
+      return fs.writeFile(newMarkdownPath, newText)
+    })
+    .then( () => fs.unlink(oldMarkdownPath))
+    .catch(err => {
```

Comment:
just remove this, errors tht propgate up should already halt the rename-skill script

Reply:
make sure the output of the script tells you which ones were dont good and not done